src/libraries/ref.proj

```diff
@@ -0,0 +1,36 @@
+<Project InitialTargets="IncludeProjectReferences">
```

Comment:
NIT: Looks like git is not considering this and ref.builds to have been the same file, which sometimes breaks the ability to check history on the file or to blame a line. When making these renames, can we try using `git mv oldFileName newFileName` so that history is preserved?

Reply:
It also makes it easier to review 😉

Reply:
just did that, thanks